manubot/cite/util.py

```diff
@@ -72,6 +120,11 @@ def is_valid_citation_string(string):
         logging.error(f'{string} → source "{source}" is not valid')
```

Comment:
We currently require a lower case source.  Could that be confusing to users, and do we want to provide a hint in the error message?  The test
```
$ manubot cite DOI:10.1038/nrg3920
## ERROR
@DOI:10.1038/nrg3920 \u2192 source "DOI" is not valid
```
prompted this.  A novice may not understand why `DOI` doesn't work.